firefox_decrypt.py

```diff
@@ -751,8 +761,9 @@ def parse_sys_args():
                         help="Export URL, username and password to pass from passwordstore.org")
     parser.add_argument("-p", "--pass-prefix", action="store", default=u"web",
                         help="Prefix for export to pass from passwordstore.org (default: %(default)s)")
-    parser.add_argument("-t", "--tabular", action="store_true",
-                        help="Output in tabular format")
```

Comment:
I would either keep this (in a backwards-compatible way) or fix the corresponding test (verifying it still does it's job. ^^).
Also: Maybe add a test for CSV?